### DISCUSSION TITLE
news/2024/01/markdown-kinda-sucks/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#3] [Giscus]
# news/2024/01/markdown-kinda-sucks/

Image stolen from Adam Michael Wood
Basic font emphasis styles First of all, I think markdown is pretty good if you want to do basic stuff like italics, bold, strikeout, underli–. Oh what?? Markdown doesn’t have a syntax for underline you say? Well what am I to do now. Oh maybe because markdown supports HTML5, I can use tags. What now? yOu cAN’t dO thaT. This doesn’t work either because this site can’t do it for some inexplicable reason even though it’s supposed to?

https://liluzibird.github.io/news/2024/01/markdown-kinda-sucks/

**Comment by armando013005** [comment]
das crazy

**Reply by liluzibird** [reply]
das crasy man

**Reply by LuisRuv21** [reply]
das crazy
